wremnants/CardTool.py

```diff
@@ -597,6 +598,112 @@ def writeForProcesses(self, syst, processes, label, check_systs=True):
         if syst != self.nominalName:
             self.fillCardWithSyst(syst)
 
+    def setSimultaneousABCD(self):
+        # Having 1 process for fakes, for each bin 3 free floating parameters, 2 normalization for lowMT and highMT and one fakerate between iso and anti iso
+        logger.info(f"Set processes for simultaneous ABCD fit")
+
+        # expected fake contribution
+        hist_fake = sum([group.hists[self.nominalName] if name == "Data" else -1*group.hists[self.nominalName] for name, group in self.datagroups.groups.items()])
```

Comment:
You could maybe use datagroups.addSummedProc here

Reply:
I investigated the method but feel it makes the code more complicated, I would leave it as it is